Dockerfile

```diff
@@ -46,7 +46,8 @@ USER lfh
 RUN mkdir -p /home/lfh/connect
 WORKDIR /home/lfh/connect
 COPY --chown=lfh:lfh ./connect ./connect
-COPY --chown=lfh:lfh Pipfile.lock logging.yaml .
+COPY --chown=lfh:lfh ./local-certs/nats-server.conf ./local-certs/server.nk ./local-certs/
```

Comment:
Could we rename the `server.nk` to `nats-server.nk`. I know this is a minor thing, but it would keep "things" together in the local cert directory. If `nats-server.nk` is considered non-standard than we can certainly stick with `server.nk`.